awesome-books-with-ES6/modules/navBar.js

```diff
@@ -0,0 +1,27 @@
+// eslint-disable-next-line import/no-cycle
+import {
+  listLink, title, table, form, contactPage, addNew, contact,
+} from '../index.js';
+
+// this section handles the page navigation
+export default function navBar() {
+  listLink.addEventListener('click', () => {
+    title.classList.remove('toggle');
```

Comment:
- [ ] Let's use ES6 syntax for our functions. Kindly change this to an arrow function